packages/react/slider/src/Slider.stories.tsx

```diff
@@ -176,6 +176,26 @@ export const WithinForm = () => {
   );
 };
 
+export const Strict = () => (
```

Comment:
Should we go ahead and add the `strictMode` flag in storybook to cover this instead?

Reply:
We can maybe do that separately because all stories will need testing thoroughly and any issues logged when we do that work. Perhaps I can update the ticket you created for it to remind whoever does it to remove it from this story?